src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/CreateReleaseNoteRequest.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api
```

Comment:
Would it be possible to put this in the same folder as the other Manage Content request classes?  e.g. AddContentBlockRequest

Reply:
Done, thanks.